src/Microsoft.Identity.Web.GraphServiceClient/Readme.md

```diff
@@ -0,0 +1,283 @@
+﻿# Microsoft.Identity.Web.GraphServiceClient
+
+With the introduction of Microsoft.Identity.Web.GraphServiceClient and Microsoft.Identity.Web.GraphServiceClientBeta
+libraries in version Microsoft.Identity.Web 2.12, you now have the choice to use either the legacy 
+Microsoft.Identity.Web.MicrosoftGraph and Microsoft.Identity.Web.MicrosoftGraphBeta NuGet packages 
+based on Microsoft Graph SDK 4.x or the new libraries based on Microsoft Graph SDK 5. 
+By keeping both options available, you can choose to migrate to the latest version of the SDK at your own pace 
+and with minimal disruption to your existing code.
+
+By migrating to Microsoft.Identity.Web.GraphServiceClient, you'll benefit from the latest features of the Microsoft Graph SDK,
+including a simplified fluent API and the ability to use both Microsoft Graph and Microsoft Graph Beta APIs in the same application.
+However, migrating from Microsoft.Identity.Web.MicrosoftGraph 2.x to Microsoft.Identity.Web.GraphServiceClient requires moving some of your code,
+as discussed in the [migration guide](#migrate-from-microsoftidentitywebmicrosoftgraph-2x-to-microsoftidentitywebgraphserviceclient).
+
+
+## Usage
+
+1. Reference Microsoft.Identity.Web.GraphServiceClient in your project.
+   ```shell
+   dotnet add package Microsoft.Identity.Web.GraphServiceClient
+   ```
+
+1. In the startup method, add Microsoft Graph support to the service collection. 
```

Comment:
But this assumes they are already using MS Graph SDK 4x, so they would already have `.AddMicrosoftGraph`, no?

Reply:
right

Reply:
The migration part is at the end of the document. This one is more the usage in general. Do you think we should invert both?